### DISCUSSION TITLE
Adding quick button

### DISCUSSION THREAD

**Discussion by korenevskiy** [0x7c13/Notepads#840] [General]
Adding quick button functionality to the title bar.
99% completed.
Help is needed.
_I suggested this idea in the discord chat.
'Soumya' answered me and told me to do PR._
PR https://github.com/JasonStein/Notepads/pull/834
![img-2021-03-28-06-17-01](https://user-images.githubusercontent.com/6898474/112750035-c7059980-8fce-11eb-863b-3326fa091148.png)
![img-2021-03-28-06-16-03](https://user-images.githubusercontent.com/6898474/112750036-c836c680-8fce-11eb-9385-3560255f94cb.png)
Dear friends and colleagues.
I have Visual Studio buggy. I can't make a Pool request.
I had to use "GitHub Desktop".
First I made the buttons in Visual Studio. But then there were problems.
I used notepad to make separate changes to the project folder, I used "GitHub Desktop" to make a Pull request.
I ask for help to conduct a check and can fix some errors.
 
Only adds the function of quick buttons. But you only need to check the click execution for the buttons.. This will allow you to create quick buttons in any sequence. The sequence is determined by the order in which the switches are turned on. And it also uses saving settings , and uses localization for Label name

1. Move button to right to buttons window.
2. Testing click buttons
3. Testing in small mode.
 
**Please add this request.
Since all these buttons are optional. And can be used for everyone to choose from.**

**Comment by 590059** [comment]
请问win10 LTSC版如何安装Notepads，谢谢！
How to install Notepads in win10 LTSC version, thank you！

**Reply by soumyamahunt** [reply]
Try installing from releases page: https://github.com/JasonStein/Notepads/releases/download/v1.4.2.0/Notepads_1.4.2.0_x86_x64.msixbundle
